libbeat/esleg/eslegclient/connection.go

```diff
@@ -135,6 +136,9 @@ func NewConnection(s ConnectionSettings) (*Connection, error) {
 	if s.Beatname == "" {
 		s.Beatname = "Libbeat"
 	}
+	if fleetmode.Enabled() {
+		s.Beatname = fmt.Sprintf("%s-Managed", s.Beatname)
```

Comment:
```suggestion
		s.Beatname = fmt.Sprintf("%s-Agent", s.Beatname)
```

Let's be a bit more direct.

Also, for `agentbeat` is the Beat name `agentbeat` in the User-Agent?

Reply:
So, as far as I can tell, `agentbeat` just imports the `RootCmd` vars from all the other beats, which means they'll connect with the name of the underlying beat that `agentbeat` is importing.